### PR TITLE
plugins: Add README.md

### PR DESCRIPTION
**Pull request by JoseMRT2004**
I added a readme that has a small overview of what each of the plugins does and included information on how to install them.👀

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+### Oh My Bash: Enhancing Your Development Experience
```
Comment (akinomyoga) [inline plugins/README.md:1]: I'm wondering about the heading level. Why doesn't this start with the `<h1>` level (i.e. single `#`)? Could you make it `<h1>`?

File: plugins/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+"Oh My Bash won't make you a tenfold developer... but it might make you feel like one."
+
+_**Embracing a suite of plugins can significantly boost your coding efficiency by automating repetitive tasks, thereby making them less monotonous and easier to execute.**_
```
Comment (akinomyoga) [inline plugins/README.md:5]: Boldfacing the entire sentence seems too strong. I'd prefer just italicization.

File: plugins/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+```bash
+plugins=(git bundler osx rake ruby)
+```
+
+**Conditional Enablement:**
+You may wish to control when or how plugins are activated. For instance, to ensure the `tmux-autoattach` plugin runs only in SSH sessions, you can employ a conditional statement checking for the `$SSH_TTY` variable. Make sure to remove the plugin from the main plugin list and use a conditional like this:
+
+```bash
+[ "$SSH_TTY" ] && plugins+=(tmux-autoattach)
```
Comment (akinomyoga) [inline plugins/README.md:21]:
```suggestion
[[ $SSH_TTY ]] && plugins+=(tmux-autoattach)
```

We want to gradually normalize the styles to use Bash constructs.

**Comment by JoseMRT2004** [comment]
**all ready ✅**

File: plugins/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+```bash
+plugins=(git bundler osx rake ruby)
+```
+
+**Conditional Enablement:**
+You may wish to control when or how plugins are activated. For instance, to ensure the `tmux-autoattach` plugin runs only in SSH sessions, you can employ a conditional statement checking for the `$SSH_TTY` variable. Make sure to remove the plugin from the main plugin list and use a conditional like this:
+
+```bash
+[ [ $SSH_TTY ] ] && plugins+=(tmux-autoattach)
```
Comment (akinomyoga) [inline plugins/README.md:21]:
```suggestion
[[ $SSH_TTY ]] && plugins+=(tmux-autoattach)
```

It's not `[ [ ... ] ]`. It's `[[ ... ]]`.

**Review by akinomyoga — CHANGES_REQUESTED** [review]
Thanks for updating. See below.

File: plugins/README.md (hunk 4 above)
Comment (akinomyoga) [reply plugins/README.md]: Thanks.

**Review by akinomyoga — DISMISSED** [review]
LGTM for now.

**Comment by JoseMRT2004** [comment]
**Entonces a darle**. _It's something like that in English So let's do it_

File: plugins/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Oh My Bash: Enhancing Your Development Experience
```
Comment (akinomyoga) [inline plugins/README.md:1]: This README is the one for OMB plugins, so "Oh My Bash" doesn't seem the correct title. For example,

```suggestion
# Oh-My-Bash Plugins: Enhancing Your Development Experience
```

**Review by akinomyoga — COMMENTED** [review]
I've again checked this.

Could you update the title of the document?

I also added commit c25b10b9be714d9660b54bf8edac0185875dc8cf. We now have a list of the plugins, so we need to notify the authors of new plugins to update the list to keep it up-to-date.

**Comment by akinomyoga** [comment]
Note: I force-pushed the branch on top of the latest master. You'll need to reset or merge the remote branch.

**Review by akinomyoga — APPROVED** [review]

**Comment by akinomyoga** [comment]
Thanks!

**Pull request merged**